closed/src/java.base/share/classes/openj9/internal/security/RestrictedSecurity.java

```diff
@@ -251,7 +416,7 @@ public static boolean configure(Properties props) {
 
                 // Print out the Trace info.
                 if (userSecurityTrace) {
```

Comment:
I don't see anywhere that sets this to true. Simliarly userSecurityAudit and userSecurityHelp are still declared.

Reply:
Those were part of the previous iteration and I missed them when cleaning up. Sorry about that. Removed all mentions and definitions.